ifrs17-template/Report/ParameterReports.ipynb

```diff
@@ -0,0 +1,521 @@
+                "\n            .ToTable()",
+                "\n            .ExecuteAsync()",
+                "\n            "
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await Report.ForObjects(allYieldCurves)",
+                "\n            .WithQuerySource(Workspace)",
+                "\n            .GroupRowsBy(x => x.GroupOfContract)",
+                "\n            .GroupColumnsBy(x => x.YieldCurveType).GroupColumnsBy(x => x.Period)",
+                "\n            .ToTable()",
+                "\n            .ExecuteAsync()"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await Report.ForObjects(singleDataNodeParameters)",
+                "\n            .WithQuerySource(Workspace)",
+                "\n            .GroupRowsBy(x => x.GroupOfContract)",
+                "\n            .GroupColumnsBy(x => x.Period)",
+                "\n            .ToTable()",
+                "\n            .ExecuteAsync()"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await Report.ForObjects(interDataNodeParameters)",
+                "\n            .WithQuerySource(Workspace)",
+                "\n            .GroupRowsBy(x => x.GroupOfContract).GroupRowsBy(x => x.LinkedDataNode)",
+                "\n            .GroupColumnsBy(x => x.Period)",
+                "\n            .ToTable()",
+                "\n            .ExecuteAsync()"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await Report.ForObjects(partnerRatings)",
+                "\n            .WithQuerySource(Workspace)",
+                "\n            .GroupRowsBy(x => x.Partner)",
+                "\n            .GroupColumnsBy(x => x.Period)",
+                "\n            .ToTable()",
+                "\n            .ExecuteAsync()"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await Report.ForObjects(partnerDefaultRates)",
+                "\n            .WithQuerySource(Workspace)",
+                "\n            .GroupRowsBy(x => x.CreditRiskRating)",
+                "\n            .GroupColumnsBy(x => x.Period)",
+                "\n            .ToTable()",
+                "\n            .ExecuteAsync()"
+            ],
```

Comment:
I would separate this NB in 2. One with dataModel and queries, one with reports.

Reply:
Being done